CLDR-18311 Add more Northern Kurdish variants

conradarcturus opened the pull request:
This ticket updates Kurdish in Iraq and Iran -- updating population estimates and splitting the previously mono-script locale (ku_TR) into one with script variants. Thanks for the help with research @roozbehp ! 

This is a big change for TC locales -- while none of the TC data is being altered -- I'm moving the ticket from Accepted to Investigate Complete because I want to make sure they review they change we are making to ku_TR (moving it to ku_Latn_TR).

CLDR-18311

- [x] This PR completes the ticket.

<!--
Thank you for your pull request.
Please see https://cldr.unicode.org/index/process for general
information on contributing to CLDR.

1. Make sure the ticket is filed at
https://unicode-org.atlassian.net/projects/CLDR/
2. Update the PR title and first line of this
message to include the ticket ID (CLDR-_____)
3. You will be automatically asked to sign the contributors’
license before the PR is accepted.
- sign: https://cla-assistant.io/unicode-org/cldr
- license: https://www.unicode.org/copyright.html#License
-->

ALLOW_MANY_COMMITS=true

roozbehp commented on common/supplemental/likelySubtags.xml:
```diff
@@ -402,6 +402,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<likelySubtag from="ku_AM" to="ku_Cyrl_AM"/>		<!--Kurdish‧?‧Armenia	➡ Kurdish‧Cyrillic‧Armenia-->
 		<likelySubtag from="ku_AZ" to="ku_Cyrl_AZ"/>		<!--Kurdish‧?‧Azerbaijan	➡ Kurdish‧Cyrillic‧Azerbaijan-->
 		<likelySubtag from="ku_GE" to="ku_Cyrl_GE"/>		<!--Kurdish‧?‧Georgia	➡ Kurdish‧Cyrillic‧Georgia-->
+		<likelySubtag from="ku_IQ" to="ku_Arab_IQ"/>		<!--Kurdish‧?‧Iraq	➡ Kurdish‧Arabic‧Iraq-->
```
I don't know for a fact if this is true (if Iraqi Kurmanji is more commonly written in Arabic than Latin), but if you ask me to decide one way or another, I'd go with what you have done here.

conradarcturus replied:
My same feeling. Not confident but for now it makes sense.

roozbehp commented on common/supplemental/supplementalData.xml:
```diff
@@ -2417,7 +2417,7 @@ XXX Code for transations where no currency is involved
 			<languagePopulation type="es" populationPercent="55" references="R1161"/>	<!--Spanish-->
 			<languagePopulation type="bzj" populationPercent="50" references="R1161"/>	<!--Belize Kriol English-->
-			<languagePopulation type="kek" populationPercent="6.5" references="R1161"/>	<!--Kekchí-->
+			<languagePopulation type="kek" populationPercent="6.5" references="R1161"/>	<!--Qʼeqchiʼ-->
```
This is unrelated. Remove.

conradarcturus replied:
This part of the file is automatically generated so I left it in. I can remove it -- the change isn't about this.

I just have 1 question, when does it get changed? If we had to file a new ticket for every missed auto-generated change, we'd file a lot of tickets...

srl295 replied:
It's a BRS item later in the release cycle.  Or, it can be done as a separate PR.  But it shouldn't be in a PR about Kurdish.

roozbehp commented on common/supplemental/supplementalData.xml:
```diff
@@ -2929,7 +2929,7 @@ XXX Code for transations where no currency is involved
 			<languagePopulation type="es" populationPercent="88" officialStatus="official" references="R1168"/>	<!--Spanish-->
 			<languagePopulation type="cak" populationPercent="8.8" officialStatus="official_regional" references="R1170"/>	<!--Kaqchikel-->
-			<languagePopulation type="kek" populationPercent="8.8" references="R1168"/>	<!--Kekchí-->
+			<languagePopulation type="kek" populationPercent="8.8" references="R1168"/>	<!--Qʼeqchiʼ-->
```
Unrelated. Remove.

srl295 replied:
Agreed. Please drop unrelated generated lines from PRs…

roozbehp commented on common/supplemental/supplementalData.xml:
```diff
@@ -3172,6 +3172,8 @@ XXX Code for transations where no currency is involved
 			<languagePopulation type="luz" populationPercent="1.2"/>	<!--Southern Luri-->
 			<languagePopulation type="lki" populationPercent="0.76" references="R1212"/>	<!--Laki-->
 			<languagePopulation type="kaa" populationPercent="0.67" references="R1199"/>	<!--Kara-Kalpak-->
+			<languagePopulation type="ckb" populationPercent="0.63" references="R1383"/>	<!--Central Kurdish-->
```
This doesn't feel right considering my personal knowledge of the Kurdish community in Iran. But I don't have any references.

conradarcturus replied:
It's much lower than the previous estimate, which cited the Iranian Statistical bureau https://www.amar.org.ir/ . The new citations are from a source cited frequently by Ethnologue, J Leclerc.

I would prefer the Census website but unfortunately the website is inaccessible for me - I get server errors. I don't know if its banned from US IPs or what the problem is. I can access it from the Wayback Machine but its very hard to navigate because I don't read Persian. I had some progress using Google Translate's photo translation feature. I eventually found a summary of the 2015 census in pdf but there was no table on languages. If you have better luck with that website and/or have a source you think is better, I'm happy to change it.

roozbehp replied:
I asked my Twitter followers for help. Let's see what they can dig up. In the meanwhile, I guess we should go with what we have.

roozbehp replied:
Everyone who has made a comment is suspicious of the new number and thinks that it is too low, but I haven't been pointed to any alternative number yet.

roozbehp commented:
I also think ku.xml should be renamed to ku_Latn.xml

conradarcturus commented:
> I also think ku.xml should be renamed to ku_Latn.xml

We keep all language-only xmls. Since the base of `ku` is still `Latn` -- `ku_Latn.xml` is still just an alias in `ku.xml`. Same deal for ff_Latn/ff, kk_Cyrl/kk...

conradarcturus commented:
Also, it looks like I need to get exemplar characters for ku_Arab. @roozbehp do you think its the same as Central Kurdish (Sorani) or different? [ئ ا ب پ ت ج چ ح خ د ر ز ڕ ژ س ش ع غ ف ڤ ق ک گ ل ڵ م ن ه ە و ۆ ی ێ]

conradarcturus commented on docs/site/downloads/cldr-48.md:
```diff
@@ -87,7 +87,7 @@ For a full listing, see [¤¤BCP47 Delta](https://unicode.org/cldr/charts/dev/de
 ### Locale Changes
 
-- TBD
+- Kurdish (Kurmanji) `ku` split from 1 locale `ku_TR` into 6 locales across 2 scripts and 4 countries: `ku_Latn_TR` is now the default but there are now entries for Syria (in Latin and Arabic writing), Iraq (in Latin and Arabic writing), and Iran (in Arabic writing). ([CLDR-18311][])
```
Preview in https://6b08cea6.cldr.pages.dev/downloads/cldr-48#locale-changes

conradarcturus replied:
![Screenshot 2025-04-15 at 14 54 27](https://github.com/user-attachments/assets/4b44c2d4-caf7-45df-a85e-7ec9fbe92770)

conradarcturus commented:
Made a few changes based on comments and test failures:
* Added exemplars for `ku_Arab.xml` by merging exemplars from `ckb.xml` and the `exemplars/main/ku_Arab.xml` files. Removed ku_Arab from the old exemplars because yay its not an official locale.
* Removed `ku_Arab_SY.xml` -- Roozbeh found some evidence for it but I cannot verify and it sounds like the new government is focused on Latin writing for Kurdish
* Added documentation to the CLDR-48 changelog

![Screenshot 2025-04-15 at 14 54 27](https://github.com/user-attachments/assets/534fdbdd-3667-41da-94a6-8ec22450700d)
